src/components/Papers/PaperLine.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react'
+import format from 'date-fns/format'
+import get from 'lodash/get'
+
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import ListItemSecondaryAction from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemSecondaryAction'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Button from 'cozy-ui/transpiled/react/Button'
+import { Icon } from 'cozy-ui/transpiled/react'
+import DotsIcon from 'cozy-ui/transpiled/react/Icons/Dots'
+import IconPdf from 'cozy-ui/transpiled/react/Icons/FileTypePdf'
+
+import papersJSON from 'src/constants/papersDefinitions.json'
+
+const PaperLine = ({ data, divider }) => {
```

Comment:
proptypes to describe can be useful ;)

Reply:
En effet, il me faut prendre cette bonne habitude maintenant en JS 😁